src/components/pages/CommunityPage/CommunityPage.tsx

```diff
@@ -24,6 +24,24 @@ export default function CommunityPage(): JSX.Element {
     <Button title={isShowMoreCmtrs ? 'Show less' : 'Show more'} variant="transparentWhite" onClick={() => setIsShowMoreCmtrs(!isShowMoreCmtrs)}/>
   );
 
+  function shuffleArray(array) {
```

Comment:
There is `lodash` function for that: https://lodash.com/docs/4.17.15#shuffle
But I do not insist ;)

Reply:
makes sense, updated. PTAL